lib/ansiblelint/file_utils.py

```diff
@@ -11,3 +22,10 @@ def normpath(path) -> str:
     """
     # convertion to string in order to allow receiving non string objects
     return os.path.relpath(str(path))
+
+
+class AFile(TypedDict):
```

Comment:
This name should be improved. Currently "a file" does not communicate the intent.
Maybe
```suggestion
class LintedFile(TypedDict):
```

Reply:
Or
```suggestion
class TargetFile(TypedDict):
```